src/app/static/src/app/components/projects/ProjectHistory.vue

```diff
@@ -9,34 +9,37 @@
         </div>
         <hr/>
         <div v-for="p in projects">
-            <div :id="`p-${p.id}`"  class="row py-2">
+            <div :id="`p-${p.id}`" class="row py-2">
                 <div class="col-md-1 project-cell">
                     <button v-b-toggle="`versions-${p.id}`" class="btn btn-xs bg-transparent shadow-none py-0">
                         <chevron-right-icon size="20" class="icon when-closed"></chevron-right-icon>
                         <chevron-down-icon size="20" class="icon when-open"></chevron-down-icon>
                     </button>
                 </div>
                 <div class="col-md-2 project-cell">
-                    {{p.name}}
+                    {{ p.name }}
                 </div>
-                <div class="col-md-1 project-cell"><small class="text-muted">{{versionCountLabel(p)}}</small></div>
-                <div class="col-md-3 project-cell">{{format(p.versions[0].updated)}}</div>
-                <div class="col-md-2 project-cell">
-                    <a @click="loadVersion($event, p.id, p.versions[0].id)" href="" v-translate="'loadLastUpdated'"></a>
+                <div class="col-md-1 project-cell"><small class="text-muted">{{ versionCountLabel(p) }}</small></div>
+                <div class="col-md-3 project-cell">{{ format(p.versions[0].updated) }}</div>
+                <div class="col-md-1 project-cell">
+                    <a @click="loadVersion($event, p.id, p.versions[0].id)" href="" v-translate="'load'"></a>
```

Comment:
Changed "Load last updated" to just "Load" to allow this column to become narrower.

Reply:
I think that's ok - we've got the project/version indicator now to show what version people are editing and I guess it's intutive that you'd load the last edited version.

Reply:
Yeah that's what I thought.